src/cursor/cur_hs.c

```diff
@@ -369,6 +685,7 @@ __wt_curhs_open(WT_SESSION_IMPL *session, WT_CURSOR *owner, WT_CURSOR **cursorp)
     WT_CURSOR *cursor;
     WT_CURSOR_HS *hs_cursor;
     WT_DECL_RET;
+    WT_ITEM *ds_key;
```

Comment:
What does `ds` mean here? A descriptive name would be better.

Reply:
it means datastore.